cogs/displayinfo.py

```diff
@@ -0,0 +1,131 @@
+import discord
+import constantvariables
+from discord.ext import commands
+from discord.ext import tasks
+from datetime import datetime
+from datetime import date
+from pymongo.mongo_client import MongoClient
+from pymongo.server_api import ServerApi
+import asyncio
+
+
+# MongoDB URI for connecting to the database
+uri = "mongodb+srv://user:user@rasenbotcluster.toeujwx.mongodb.net/?retryWrites=true&w=majority"
+
+# Create a new client and connect to the MongoDB server
+mongoClient = MongoClient(uri, server_api=ServerApi('1'))
```

Comment:
We can make the connection in `main.py` and then comment where the database is used. So far, only for this cog.